android-o/AndroidPlayLocation/LocUpdPendIntent/LocUpdPendIntent/Properties/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.xamarin.LocUpdPendIntent">
-	<uses-sdk android:targetSdkVersion="26" />
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.xamarin.LocUpdPendIntent" android:installLocation="auto">
+	<uses-sdk android:targetSdkVersion="23" />
```

Comment:
Both the manifest changes here seem unrelated, can we revert them?

Reply:
Reverted.